.circleci/config.yml

```diff
@@ -19,12 +23,13 @@ jobs:
           command: |
             python -m virtualenv env
 
-      - run: &install-dependencies
+      - run:
           name: install dependencies
           command: |
             . env/bin/activate
             python --version
             pip install -r requirements.txt -r tests/requirements.txt
+            pip install 'dimod<< parameters.dimod-version >>'
```

Comment:
nit/code de-dup: we could move this into a separate step e.g. patch or constrain dependencies, so we don't have to repeat the install steps below for mac.